<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Preview_Screen</class>
 <widget class="QWidget" name="HNS_Preview_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PB_Preview_Edit">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>90</y>
     <width>162</width>
     <height>107</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px
}</string>
   </property>
   <property name="text">
    <string>EDIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Preview_SaveToLibrary">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>162</width>
     <height>107</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px
}</string>
   </property>
   <property name="text">
    <string>SAVE TO
LIBRARY</string>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Preview_Preview" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>620</width>
     <height>350</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Preview_Schedule">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>490</y>
     <width>283</width>
     <height>107</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>SAVE AND SCHEDULE
NOW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Preview_Back">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>610</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Preview_Run">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>490</y>
     <width>283</width>
     <height>107</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>SAVE AND RUN
NOW</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1A3760</string>
   </property>
   <property name="text">
    <string>PREVIEW MESSAGE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNS_Message_Preview</class>
   <extends>QWidget</extends>
   <header>hns_message_preview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
